griptape/tasks/image_inpainting_task.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from typing import Callable, Tuple
+
+from attr import define, field
+
+from griptape.artifacts import ImageArtifact, TextArtifact
+from griptape.tasks import BaseImageGenerationTask, BaseTask
+from griptape.utils import J2
+
+
+@define
+class ImageInpaintingTask(BaseImageGenerationTask):
```

Comment:
This and the other image input Tasks should probably subclass from some sort of Image Input Base class. Though I don't know how to deal with the differences in the in/outpainting Tasks vs the variation Task.

Reply:
Spun off as #526